services/collaboration/pkg/service/grpc/v0/service.go

```diff
@@ -123,10 +124,12 @@ func (s *Service) OpenInApp(
 		viewAppURL = editAppURL
 	}
 
-	wopiSrcURL := url.URL{
-		Scheme: s.config.HTTP.Scheme,
-		Host:   s.config.HTTP.Addr,
-		Path:   path.Join("wopi", "files", fileRef),
+	wopiSrcURL, err := url.Parse(strings.Replace(s.config.WopiApp.WopiSrc, "{fileid}", fileRef, 1))
```

Comment:
A couple of things:
I guess we might want to allow pointing the WopiSrc to a different server... but why? I'd expect the URLs always to point to this server.
Having a variable `{fileid}` in the URL doesn't seem a good idea. It will likely confuse the admin and we'll have to manage that scenario. It seems better for the admin to provide the server's URL so we can build the rest on our own.